### DISCUSSION TITLE
Linux 下识别第三次就崩溃

### DISCUSSION THREAD

**Discussion by 2284445882** [PaddlePaddle/PaddleOCR#14427] [Q&A]
捕获的异常--》RuntimeError: could not execute a primitive
windows下不会复现，是什么原因可能导致这种情况

**Comment by GreatV** [comment]
可以试试dev版本的paddlepaddle，应该已经修复了。

**Reply by 2284445882** [reply]
我是用的C++版本的SDK  一样吗

**Reply by GreatV** [reply]
这个是paddle框架的问题，可以试试换成dev版本的c++ paddle inference

**Reply by GreatV** [reply]
或者搜索一下历史issue，里面也有一些建议

**Reply by 2284445882** [reply]
好的 谢谢
